sqlglot/dialects/doris.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+import typing as t
+
+from sqlglot import exp, generator
+from sqlglot.dialects.dialect import (
+    approx_count_distinct_sql,
+    arrow_json_extract_sql,
+    rename_func,
+)
+from sqlglot.dialects.mysql import MySQL
+from sqlglot.helper import seq_get
+
+
+def _to_date_sql(self: generator.Generator, expression: exp.TsOrDsToDate) -> str:
+    this = self.sql(expression, "this")
+    self.format_time(expression)
+    return f"TO_DATE({this})"
+
+
+def _time_format(
+    self: generator.Generator, expression: exp.UnixToStr | exp.StrToUnix
+) -> t.Optional[str]:
+    time_format = self.format_time(expression)
+    if time_format == Doris.TIME_FORMAT:
+        return None
+    return time_format
+
+
+class Doris(MySQL):
```

Comment:
Because doris is compatible with most of the syntax of mysql, but with the development of the doris community, some newly added syntax is different from the usage of mysql. For the convenience of maintenance later, the doris dialect is added.